src/sentry/receivers/sentry_apps.py

```diff
@@ -95,6 +122,30 @@ def send_workflow_webhooks(
             data=data,
         )
 
+    if features.has("organizations:sentry-functions", organization, actor=user):
+        from sentry.models import SentryFunction
+
+        data["user"] = serialize(User.objects.get(id=user.id), user, UserSerializer())
+        data["issue"] = serialize(Group.objects.get(id=issue.id))
+        for fn in SentryFunction.objects.filter(organization=organization).all():
+            if "issue" not in fn.events:
+                continue
+            from google.cloud import pubsub_v1
+
+            from sentry.utils import json
+
+            google_pubsub_name = "projects/hackweek-sentry-functions/topics/fn-" + fn.external_id
```

Comment:
@vpandiarajan20 I think you should make a function to generate this since we have this code repeated